tests/flags_to_string_int/flags_to_string_int.c

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#include <stdlib.h>
```

Comment:
>clude <stdlib.h> [](http://example.com/codeflow?start=3&length=16)

Likely missing inttypes. Also not sure whether stdlib is actually needed

Reply:
Removed them all. "if it compiles it is good"

Reply:
Hope the gate agrees to that.